### PR TITLE
Teams workaround

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-18583

## Description
added a workaround for the problem of not creating a communication when the bot is added to a team.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 1]
```diff
@@ -1613,12 +1616,16 @@ def messages() -> Response:
             if event_type == 'teamMemberAdded':
                 demisto.info('New Microsoft Teams team member was added')
                 member_added_handler(integration_context, request_body, channel_data)
+                demisto.debug(f'Updated integration context: {json.dumps(get_integration_context().get("teams"))}')
```
Comment (DeanArbel) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:1619]: Please provide a message that'll be clear when looking at the logs. e.g.
```
Updated team members in the integration context. Current saved members: {json.dumps(get_integration_context().get("teams"))}
```

**Review by DeanArbel — APPROVED** [review]
Looks good, just one small note (see below).

**Review by DeanArbel — APPROVED** [review]
Nice

**Pull request merged**